main/src/cgeo/geocaching/utils/TextUtils.java

```diff
@@ -420,6 +428,29 @@ public static String replaceAll(final String text, final String startToken, fina
         return result;
     }
 
+    @NonNull
+    public static List<String> getAll(final String text, final char delimiter, final  Boolean ignoreEmpty) {
```

Comment:
I am not sure I understand what this method is doing. Is there a Unit test for it?

Reply:
I remove it with String.split ;-)